Modules/TestQuestionPool/classes/tables/class.ilUnitCategoryTableGUI.php

```diff
@@ -34,10 +34,12 @@ public function __construct(ilUnitConfigurationGUI $controller, $cmd)
         $this->setDefaultOrderDirection('category');
         $this->setDefaultOrderDirection('ASC');
 
-        if ($this->getParentObject()->isCRUDContext()) {
-            $this->addMultiCommand('confirmDeleteCategories', $this->lng->txt('delete'));
-        } else {
-            $this->addMultiCommand('confirmImportGlobalCategories', $this->lng->txt('import'));
+        if ($DIC->rbac()->system()->checkAccess('write', $DIC->workflowEngine()->internal()->request()->getRefId())) {
```

Comment:
AFAIK this tables is used in different context:

1. Global administration
2. In a concrete test object in the repository

Depending on the context, `ilRbacSystem` (`$DIC->rbac()->system()`) or `ilAccessHandler` (`$DIC->access()`) MUST be used for access checks.